docs/src/extend/index.md

```diff
@@ -21,34 +21,34 @@ In order to work with ESLint as a developer, it's recommended that:
 
```

Comment:
This page currently contains a mix of links in `contribute` and `extend`. Are you planning on splitting this up in a future PR?

Reply:
yes, that's on my radar, but explicitly wrote down as part of new issue https://github.com/eslint/eslint/issues/16720

Reply:
Sounds good.